signac/common/config.py

```diff
@@ -5,195 +5,107 @@
 
 import logging
 import os
-import stat
 
-from ..common.deprecation import deprecated
-from ..version import __version__
 from .configobj import ConfigObj, ConfigObjError
 from .errors import ConfigError
 from .validate import cfg, get_validator
 
 logger = logging.getLogger(__name__)
 
-DEFAULT_FILENAME = ".signacrc"
-CONFIG_FILENAMES = [DEFAULT_FILENAME, "signac.rc"]
-HOME = os.path.expanduser("~")
-CONFIG_PATH = [HOME]
-FN_CONFIG = os.path.expanduser("~/.signacrc")
+PROJECT_CONFIG_FN = os.path.join(".signac", "config")
+USER_CONFIG_FN = os.path.expanduser(os.path.join("~", ".signacrc"))
 
+# TODO: Consider making this entire module internal and removing all its
```

Comment:
Future work: It looks like the only public part of the API is the `Config` class. We may be able to separate this, or make `Config` a private class (not sure if that would matter, since it's immutable).